packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -67,10 +67,12 @@ final DecorationTween _kGradientShadowTween = new DecorationTween(
 ///
 /// See also:
 ///
-///  * [MaterialPageRoute] for an adaptive [PageRoute] that uses a platform
+///  * [MaterialPageRoute], for an adaptive [PageRoute] that uses a platform
 ///    appropriate transition.
```

Comment:
platform-appropriate

Reply:
Added the hyphen and wrapped; it's not biggie that the line is shorter.